docs/src/tutorials/child-device-config-mngmnt.md

```diff
@@ -0,0 +1,297 @@
+3. On the right side, under configurations, click on the in *Step 1 Startup / Bootstrap of the child device* created config file (`config1`) in the column: `DEVICE-SUPPORTED CONFIGURATIONS` click on the config file (`config1`) and click on `Get snapshot from device`. This will trigger a config snapshot requests via MQTT.
+4. Send an “executing” operation status update to acknowledge the receipt of the request via MQTT. After receiving of the request, the agent must send an "executing" MQTT status message as follows:
+
+**Topic:**
+
+`tedge/{child-d}/commands/res/config_snapshot`
+
+**Payload**:
+
+```json
+{
+    "status": "executing",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+
+Example:
+
+`
+mosquitto_pub -h 127.0.0.1 -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "executing", "path": "/home/pi/config/config1", "type": "config1"}'
+`
+
+
+5. Upload the requested config file to the URL received in the request via HTTP. After sending this 'executing' status message, the agent must upload the requested configuration file content to the url received in the request with an HTTP PUT request.
+
+Example:
+
+`
+curl -X PUT --data-binary @/home/pi/config/config2 http://127.0.0.1:8000/tedge/file-transfer/child1/config_snapshot/config1
+`
+
+6. Send a “successful” operation status update via MQTT. Once the upload is complete, send a "successful" MQTT status message as follows:
+
+**Topic:**
+
+`tedge/{child-d}/commands/res/config_snapshot`
+
+**Payload**:
+
+```json
+{
+    "status": "successful",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+
+Example:
+
+```
+mosquitto_pub -h localhost -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "successful", "path": "/home/pi/config/config1", "type": "config1"}'
+```
+
+### Step 3 Handle config update request from thin-edge.io
+
+Handling config update requests from thin-edge.io is a 5-step process:
+
+1. Subscribe to, and receive config update requests via MQTT. In Cumulocity IoT, go to `device management` ->  `Management` -> `Configuration repository` -> `Add configuration snapshot` and fill in the fields in the pop-up . Upload a simple text file will be sufficient as an example.Give it the name config1.
+2. Click on `Devices` -> `All Devices`, then click on your device. Then click on  `Child devices` ->  then click on the in *Step 1 Startup / Bootstrap of the child device* defined child device (child1) and then click on `Configuration`.
+3. On the right side, under configurations, click on the in *Step 1 Startup / Bootstrap of the child device* created config file (`config1`) in the column: `DEVICE-SUPPORTED CONFIGURATIONS` click on the config file (`config1`)
```

Comment:
The same mistakes in the phrasing of these sentences, highlighted in the earlier "config snapshot" section. is repeated here as well. Please correct those.